docs/devguide/metricbeat-devguide.asciidoc

```diff
@@ -18,7 +18,6 @@ This following topics describe how to contribute to Metricbeat by adding metrics
 * <<creating-metricsets>>
 * <<metricset-details>>
 * <<creating-metricbeat-module>>
```

Comment:
Ok, I see we still have references here to the docs about creating metricsets, does it refer to `docs/devguide/create-metricset.asciidoc`, or there are other documents somewhere else?